_OmsiHookExamples/Multiplayer_Client/OMSIClient.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Multiplayer_Client;
+using System;
+using System.Text;
+using Telepathy;
+
+class OMSIClient
+{
+    static void Main(string[] args)
+    {
+        Client client = new Client();
+        GameClient gameClient = new GameClient();
+        client.Connect("127.0.0.1", 1337);
+
+
+        while (true)
+        {
+            Telepathy.Message msg;
+            while (client.GetNextMessage(out msg))
```

Comment:
Just realised that the version of Telepathy on NuGet is really out of date (from 2019)...  
Latest build is here: https://github.com/MirrorNetworking/Telepathy/actions/runs/10029006044

Anyway, is this a blocking method? If so, why do you thread sleep at the end of this method?